examples/compiled/a_test.vg.json

```diff
@@ -0,0 +1,128 @@
+{
+  "$schema": "https://vega.github.io/schema/vega/v4.json",
+  "autosize": "pad",
+  "padding": 5,
+  "height": 200,
+  "data": [
+    {
+      "name": "source_0",
+      "url": "https://vega.github.io/vega-datasets/data/seattle-weather.csv",
+      "format": {"type": "csv"}
```

Comment:
@domoritz @kanitw Is there a way to add `"parse": {"date": "date"}` to `format` when there is `timeFormat` specified in `axis` like in examples/specs/a_test.vl.json?

Reply:
That's right.